python/activator/middleware_interface.py

```diff
@@ -181,19 +206,11 @@ def ingest_image(self, oid: str) -> None:
             image bucket.
         """
         _log.info(f"Ingesting image '{oid}'")
-        run = f"{self.instrument}/raw/all"
-        cmd = [
-            "butler",
-            "ingest-raws",
-            "-t",
-            "copy",
-            "--output_run",
-            run,
-            self.repo,
-            f"gs://{self.image_bucket}/{oid}",
-        ]
-        _log.debug(str(cmd))
-        # subprocess.run(cmd, check=True)
+        file = f"{self.prefix}{self.image_bucket}/{oid}"
+        result = self.rawIngestTask.run([file])
+        # We only ingest one image at a time.
+        assert len(result) == 1
```

Comment:
This won't "crash the process", it will raise. Until we have a better handle on how we plan to do exception handling (I'm thinking we'll want several custom exceptions?), this seems equivalent to an `if len(result) != 1: raise RuntimeError()`.

Reply:
Then please include a TODO comment; given [the conditional way `assert` raises](https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement), we don't want to leave it in by accident.